src/components/SortSearch.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react'
+
+function SortSearch(props) {
+    return (
+        <div className="btn-group" role="group" aria-label="Basic example">
+            SEARCH BY 
+            <button type="button" className="btn btn-sort_toggle" onClick={props.sortTitles}>Title</button>
```

Comment:
you can create a separate component for button to avoid writing of the the same classNames and etc

Reply:
and use it as <Button onClick={ handler } />